src/modules/sdlog2/sdlog2.c

```diff
@@ -158,6 +160,7 @@ static const int MIN_BYTES_TO_WRITE = 512;
 
 static bool _extended_logging = false;
 
+static const char *mountpoint = "/fs/microsd";
```

Comment:
Could we concatenate the strings here at compile time to avoid code / path duplication? Like in this [example](http://stackoverflow.com/questions/14727003/c-alternative-to-macro-for-compile-time-string-literal-concatenation)